### PR TITLE
feature(1033081): Adding the contents for the new and enhanced feature in the UG for assist view

### PR DESCRIPTION
**Pull request by AbarnaMariyappan-SF5045**
### Feature description
Added the contents for the UG documentation for the new and enhanced features in Assist-View.

### Code Studio usage(Mandatory)
* Code Studio used in this PR/MR?
    - [ ] Yes
    - [x] No
* If `Yes`: Primary use (choose one)
    - [ ] Generate new code
    - [ ] Refactor/improve existing code
    - [ ] Tests
    - [ ] Bug fix / debugging help
    - [ ] Docs / comments
    - [ ] Review assistance (explanations/summaries)
    - [ ] Other: 

* MD File

* Outcome
    - [ ] Saved time
    - [ ] Neutral
    - [ ] Cost time
* If “Cost time” explain in short (1 or 2 lines):

### Requirement and specification document
* [ ] API Review (Task link: _________________)
* [x] Azure Task with detailed feature information (Task link: https://dev.azure.com/EssentialStudio/Ej2-Web/_workitems/edit/1003807)
* [ ] Not applicable

### Feature scope and impact
* [ ] Core functionality (high impact)
* [x] Enhancement to existing feature
* [ ] New standalone feature
* [ ] Experimental feature

### Feature matrix documentation
* [ ] Feature matrix document updated [Centralized location](https://syncfusion.sharepoint.com/:f:/r/sites/Blazor/Shared%20Documents/Feature%20Matrix%20-%20Documents?csf=1&web=1&e=OGX1td)
* [x] Not applicable

### Testing coverage
* [ ] Tested against feature matrix
* [x] Edge cases and error scenarios verified
* [ ] Cross-browser testing completed

### Test cases and automation
* [x] Test cases documented and attached to this PR
* [ ] BUnit tests added (PR link: _________________)
* [ ] Playwright tests added (PR link: _________________)
* [ ] Manual testing only (justification: _________________)
* [ ] Not applicable

### Test bed sample location
Provide the test bed sample location for code reviewers to verify the feature's behavior.
 
### Platform verification
* [ ] Verified in Blazor Server
* [ ] Verified in Blazor WASM
* [x] Not applicable

### Related components
* [ ] Feature applicable to other components (Tasks created: _________________) (tagged "needs-attention-coreteam")
* [x] Not applicable to other components

### Breaking changes
* [ ] Contains breaking changes - provide migration guidance (added "breaking-change" label)
* [x] No breaking changes

### Performance verification
* [ ] Memory usage verified (no leaks)
* [ ] Rendering performance verified
* [x] Not applicable

### AI Usage
Code Studio used in this PR/MR?
* [x] Yes
* [ ] No

If Yes: Primary use (choose one)
* [ ] Generate new code
* [x] Refactor/improve existing code
* [x] Tests
* [ ] Bug fix / debugging help
* [ ] Docs / comments
* [ ] Review assistance (explanations/summaries)
* [ ] Other
* [ ] NA

### Outcome
* [ ] Saved time 
* [x] Neutral
* [ ] Cost time
* [ ] If “Cost time”: (________________1-line explanation)
* [ ] NA

### Reviewer Checklist
* [ ] Feature matrix documentation verified
* [ ] Test coverage verified
* [x] Sample implementation reviewed
* [x] Code meets standards and best practices
* [ ] C# implementation used appropriately (minimal JavaScript)

### CODE REVIEW

File: ej2-asp-core-mvc/ai-assistview/EJ2_ASP.NETCORE/chain-of-thoughts.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+| Property | Description |
+|---|---|---|---|
+| `TextBlock` | Unique identifier for the block, used for collapsing/expanding state. |
+| `ToolBlock` | Identifies this block as a thinking block. Required. |
+| `ThinkingBlock` | Heading text shown in the collapsible header. |
+
+## Configure the thinking block
+ 
+You can use the `Thinking` block type in the blocks array of the a`ddPromptResponse` method to dynamically push blocks including thinking blocks into the component at runtime. Pass an object containing a blocks array, and set the second argument `isFinalUpdate` to false during streaming and true for the final update
```
Comment (IndrajithSrinivasan) [inline ej2-asp-core-mvc/ai-assistview/EJ2_ASP.NETCORE/chain-of-thoughts.md:27]: add the method name properly @MohamedSaifudeen-SF5064

**Review by IndrajithSrinivasan — APPROVED** [review]
